packages/manager/apps/pci-object-storage/src/pages/object-storage/storage/_components/StorageListColumns.component.tsx

```diff
@@ -18,9 +18,10 @@ import storages, {
   FormattedStorage,
   ObjectStorageTypeEnum,
 } from '@/types/Storages';
-import { octetConverter } from '@/lib/bytesHelper';
+import { localeBytesConverter } from '@/lib/bytesHelper';
 import { RegionTypeBadge } from '@/components/region-type-badge/RegionTypeBadge.component';
 import RegionWithFlag from '@/components/region-with-flag/RegionWithFlag.component';
+import { useLocale } from '@/hooks/useLocale';
```

Comment:
Maybe it would be easier to create a hook `useLocalBytesConverter` that would get the local under the hood. This would reduce the complexity by avoiding using useLocale everywhere:

```
import { useBytes } from '@/hooks/useBytes';
const bytesConverter = useBytes();

...

bytesConverter(row.original?.objectsSize, false, 2)
```